app/resources/logout.html.template

```diff
@@ -0,0 +1,10 @@
+{% extends "app-base.html.template" %}
```

Comment:
The main goal of this page is to show an error that the user is not an admin. Then the name "logout.html.template" is a bit strange. How about e.g., "not_admin_error.html.template"?